pkg/utils/secrets/manager/manager.go

```diff
@@ -165,6 +174,10 @@ func (m *manager) initialize(ctx context.Context, secretNamesToTimes map[string]
 
 	// Check if the secrets must be automatically renewed because they are about to expire.
 	for name, secret := range nameToNewestSecret {
+		if isCASecret(secret.Data) && rotation.NoCASecretAutoRotation {
+			continue
```

Comment:
We could even return an error here and set shoots to failed if there are CAs that are about to expire soon to make this very visible.

Reply:
If we decide to make the validity of CAs visible then we should probably rather think about a `condition` or our new `.status.credentials` structure in the `ShootStatus`. Setting shoots to `Failed` just because a CA is expiring soon is probably not a good idea. Either way, this code here is the wrong place for such shoot-specific actions. Consequently, I would not return an error here and just adhere to the config (which states skipping CA secrets/not considering them).

Reply:
Agree with what you said. Let's add this information to the shoot status then.

Reply:
I've added this to the task list in https://github.com/gardener/gardener/issues/3292